src/main/java/com/faforever/client/teammatchmaking/MatchmakingQueueItemController.java

```diff
@@ -161,4 +163,14 @@ public void onJoinLeaveQueueClicked(ActionEvent actionEvent) {
   public void showMapPool(ActionEvent actionEvent) {
     eventBus.post(new ShowMapPoolEvent(queue));
   }
+
+  public void addIsInMatchmakerTabListener(BooleanProperty isInMatchmakerTab) {
+    JavaFxUtil.addListener(isInMatchmakerTab, observable -> {
+      if (isInMatchmakerTab.get()) {
+        queuePopTimeUpdater.play();
+      } else {
+        queuePopTimeUpdater.pause();
+      }
+    });
```

Comment:
Rather than listeners for every queue item I think it would be better to have a start and stop timer method. Then the controller that displays the timers calls this start and stop method when it is displayed/hidden